style.css

```diff
@@ -91,16 +109,449 @@
+    background-repeat: no-repeat;
+    background-size: cover;
+    border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  }
+
+  .mac4 {
+    position: absolute;
+    display: flex;
+    width: 100%;
+    height: 186px;
+    left: 0;
+    top: 1813px;
+    background: url(/Asset/mac4.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  }
+
+  .mac5 {
+    position: absolute;
+    display: flex;
+    width: 100%;
+    height: 186px;
+    left: 0;
+    top: 2179px;
+    background: url(/Asset/mac5.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  }
+
+  .mac6 {
+    position: absolute;
+    display: flex;
+    width: 100%;
+    height: 186px;
+    left: 0;
+    top: 2546px;
+    background: url(/Asset/mac6.svg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    border: 0.333333px solid rgba(255, 255, 255, 0.1);
+  }
+
+  .grid-item h2 {
+    position: absolute;
+    width: 264px;
+    height: 16px;
+    left: 20px;
+    top: 20px;
+    font-family: "Poppins", sans-serif;
+    font-style: normal;
+    font-weight: 300;
+    font-size: 20px;
+    line-height: 80%;
+
+    /* identical to box height, or 16px */
+
+    color: #fff;
+  }
+
+  ul {
+    list-style-type: none;
+    display: flex;
+  }
+
+  .html {
+    position: absolute;
+    display: flex;
+    width: 69px;
+    height: 20px;
+    left: 20px;
+    top: 60px;
+    background: #98dede;
+  }
+
+  li.html {
+    font-size: 12px;
+    color: #333;
+    font-weight: bold;
+    font-family: Arial, sans-serif;
+    padding: 3px 0 0 3px;
+  }
+
+  .ruby {
+    position: absolute;
+    display: flex;
+    width: 100px;
+    height: 20px;
+    left: 99px;
+    top: 60px;
+    background: #98dede;
+  }
+
+  li.ruby {
+    font-size: 12px;
+    color: #333;
+    font-weight: bold;
+    font-family: Arial, sans-serif;
+    padding: 3px 0 0 3px;
+  }
+
+  .javas {
+    position: absolute;
+    display: flex;
+    width: 74px;
+    height: 20px;
+    left: 210px;
+    top: 60px;
+    background: #98dede;
+  }
+
+  li.javas {
+    font-size: 12px;
+    color: #333;
+    font-weight: bold;
+    font-family: Arial, sans-serif;
+    padding: 3px 0 0 3px;
+  }
+
+  .stp {
+    position: absolute;
+    display: flex;
+    width: 178px;
+    height: 44px;
+    left: 20px;
+    top: 95px;
+    border: 1px solid #fff;
+  }
+
+  .buttn {
+    position: absolute;
+    width: 178px;
+    height: 44px;
+    left: 20px;
+    top: 95px;
+    background-color: transparent;
+    border: 1px solid white;
+    font-size: 18px;
+    color: white;
+    padding: 0 30px 0 0;
+    background-image: url(/Asset/union.svg);
+    background-size: 25px;
+    background-repeat: no-repeat;
+    background-position: center right;
+  }
+
+  .about {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    height: 100vh;
+  }
```

Comment:
- [ ] As the mandatory requirement by @nicupop729 **to fix the UI** the about section is overlapping the works section, please 🙏🏽 fix this behavior to match the Figma template. 👇🏽 

**Example**

https://user-images.githubusercontent.com/58642949/220820620-d9bdb08a-b950-4cce-b8e6-25c65c1715e1.mp4

**Your design**

https://user-images.githubusercontent.com/58642949/220820258-504f4ef3-b1cf-47f8-b926-8a70a3243a7c.mp4

Reply:
Hello the about section was perfect when I add the grid element inside section. It comes down the screen. So I fix it back now Everything is okay. Why do I need to move h1 about